roles/xcode/tasks/main.yml

```
@@ -5,17 +5,49 @@
     user_install: false
     executable: "$HOME/.rbenv/shims/gem"
 
-- name: Install xcode version based on xcode_version variable, it could take a while. About an hour, so be patient.
+- name: Install xcode '{{ xcode_version }}', it could take a while. About an hour, so be patient.
   shell: bash -l -c "$HOME/.rbenv/shims/xcversion install '{{ xcode_version }}'"
```

So I learned this cool thing last week
```suggestion
  shell: xcversion install '{{ xcode_version }}'
  args:
    executable: "$HOME/.rbenv/shims/xcversion"
```

---

or it can be
```
shell: "$HOME/.rbenv/shims/xcversion install '{{ xcode_version }}'"
  args:
    executable: /bin/bash
```

---

we know why this doesn't work :)

---

yes, now we know